contracts/scripts/DeployDevnet.s.sol

```diff
@@ -84,7 +84,7 @@ contract DeployDevnet is Script {
     }
 
     function deployPBHSignatureAggregator() public {
-        pbhSignatureAggregator = address(new PBHSignatureAggregator(pbhEntryPoint));
+        pbhSignatureAggregator = address(new PBHSignatureAggregator(pbhEntryPoint, worldIdGroups));
```

Comment:
`WorldIdGroups` is for the router contract. Lets use `WorldId` instead to avoid confusion.